### DISCUSSION TITLE
multiple products named 'SwiftProtobuf' in: 'swift-protobuf', 'walletcore'

### DISCUSSION THREAD

**Discussion by enchhuang** [trustwallet/wallet-core#2771] [Q&A]
1. Firebae package (github.com/firebase/firebase-ios-sdk)
2. WalletCore (github.com/trustwallet/wallet-core)
When using these two packages at the same time, a conflict occurs and cannot be used.

Showing Recent Messages
multiple products named 'SwiftProtobuf' in: 'swift-protobuf', 'walletcore'

Showing Recent Messages
multiple targets named 'SwiftProtobuf' in: 'swift-protobuf', 'walletcore'; consider using the `moduleAliases` parameter in manifest to provide unique names

**Comment by hewigovens** [comment]
This can be solved later when we refactor the build system cc @Milerius
